### PR TITLE
Remove superfluous module hierarchy

### PR DESCRIPTION
**Pull request by nnmm**

### CODE REVIEW

**Comment by nnmm** [comment]
ptal @krzesi-mir

File: src/octree/tests.rs [hunk 1]
```diff
@@ -0,0 +1,235 @@
+
```
Comment (krzesi-mir) [inline src/octree/tests.rs]: nit: remove empty line

**Comment by nnmm** [comment]
I guess it doesn't pick up on the move because one layer of indentation is gone

File: src/octree/tests.rs (hunk 1 above)
Comment (nnmm) [reply src/octree/tests.rs]: Done

**Comment by krzesi-mir** [comment]
> I guess it doesn't pick up on the move because one layer of indentation is gone

probably

**Pull request merged**